pkg/api/validation/events.go

```diff
@@ -26,6 +26,7 @@ func ValidateEvent(event *api.Event) validation.ErrorList {
 	allErrs := validation.ErrorList{}
 	// There is no namespace required for node.
 	if event.InvolvedObject.Kind == "Node" &&
+		event.Namespace != api.NamespaceDefault &&
```

Comment:
// Older client code accidentally set event.Namespace to api.NamespaceDefault, so we accept that too, but "" is preferred.